kubernetes_practice/resources/server.py

```diff
@@ -0,0 +1,46 @@
+import http.server as server
+import logging
+import sys
+
+"""
```

Comment:
Put this docstring inside the class. See the Google's python guide for the rationales and details.